src/detect-tls-version.c

```diff
@@ -163,11 +168,9 @@ static DetectTlsVersionData *DetectTlsVersionParse (DetectEngineCtx *de_ctx, con
     if (ret > 1) {
         char ver_ptr[64];
         char *tmp_str;
-        pcre2len = sizeof(ver_ptr);
-        res = pcre2_substring_copy_bynumber(
-                parse_regex.match, 1, (PCRE2_UCHAR8 *)ver_ptr, &pcre2len);
+        res = pcre_copy_substring((char *)str, ov, MAX_SUBSTRINGS, 1, ver_ptr, sizeof(ver_ptr));
```

Comment:
You should now use PCRE2 which got merged in master, and no longer PCRE1...
Basically, that means you should leave untouched `DetectParsePcreExec` and such... and your PR should work